sound/soc/sof/ipc.c

```diff
@@ -391,11 +388,8 @@ int snd_sof_ipc_reply(struct snd_sof_dev *sdev, u32 msg_id)
 EXPORT_SYMBOL(snd_sof_ipc_reply);
 
 /* DSP firmware has sent host a message  */
-static void ipc_msgs_rx(struct work_struct *work)
+void snd_sof_ipc_msgs_rx(struct snd_sof_dev *sdev)
```

Comment:
need a prototype in sof-priv.h ?

Reply:
it is already there, this isn't an addition of a new function. This function used to schedule an rx work, now it actually does the RX.